mypyc/ops.py

```diff
@@ -335,8 +335,10 @@ def to_str(self, env: 'Environment') -> str:
 class AssignmentTargetTuple(AssignmentTarget):
     """x, ..., y as assignment target"""
 
-    def __init__(self, items: List[AssignmentTarget]) -> None:
+    def __init__(self, items: List[AssignmentTarget],
+                 starred: Optional[AssignmentTarget] = None) -> None:
         self.items = items
+        self.starred = starred
```

Comment:
I'd suggest storing the (optional) index of the starred element instead of the element itself. It will simplify the code gen a bit I think.